libyara/include/yara/lexer.h

```diff
@@ -127,7 +127,7 @@ void yyerror(
 void yywarning(
     yyscan_t yyscanner,
     const char *message_fmt,
-    ...);
+    ...) YR_PRINTF(2, 3);
```

Comment:
Something like `YR_PRINTF_LIKE` is more self-explanatory. `YR_PRINTF` suggests that it's a wrapper around `printf` similarly as how `yr_malloc` wraps `malloc`.